applications/StructuralMechanicsApplication/structural_mechanics_application.h

```diff
@@ -171,8 +171,6 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
-KRATOS_API_EXTERN template class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) AdjointSolidElement<TotalLagrangian>;
```

Comment:
Not quite sure why this is in the application header? Can we safely move it to where its suppose to be?

Reply:
I would say so, seems like you moved it to (the IMO) correct place

any objections @msandre ?

Reply:
ping @msandre